Apache-Beam-Jobs.md

```diff
@@ -639,6 +639,31 @@ Even though it is possible to use NDB functions directly, they should not be use
 
 All of the aforementioned classes are already used in the codebase so you can look for examples.
 
+### Use `get_package_file_contents` for accessing files
```

Comment:
Please update the Table of contents too.

Reply:
Done.